tests/test_mission.py

```diff
@@ -884,3 +886,119 @@ def test_smoke_required_modules(self) -> None:
         m2 = dcs.mission.Mission()
         m2.load_file(saved_miz)
         self.assertEqual(m.required_modules, m2.required_modules)
+
+    def test_big_formation_action_leader(self) -> None:
+        m_name = "tests/missions/big-formation.miz"
+        m = dcs.mission.Mission()
+        m.load_file(m_name)
+
+        assert isinstance(m.coalition['blue'].country("Combined Joint Task Forces Blue")
+                          .plane_group[0].points[0].tasks[5], WWIIFollowBigFormation)
+
+        task = m.coalition['blue'].country("Combined Joint Task Forces Blue").plane_group[0].points[0].tasks[5]
+
+        self.assertNotIn("groupId", task.params)
+        self.assertNotIn("lastWptIndex", task.params)
+        self.assertEqual(task.params["formationType"], WWIIFollowBigFormation.FormationType.COMBAT_BOX_FOR_OPEN_FORMATION)
+        self.assertEqual(task.params["pos"], WWIIFollowBigFormationPosition(0, 0, 0))
+        self.assertTrue(task.params["lastWptIndexFlagChangedManually"])
+        self.assertEqual(len(task.params), 7)
```

Comment:
Alternatively you could do something like:

```python
self.assertEqual(
    {
        "formationType": ...
        ...
    },
    task.params,
)
```

Up to you.

Reply:
I'll leave it like that for now, but understand the proposal and will refactor in the future.